third_party/com_github_census_instrumentation_opencensus_proto.patch

```diff
@@ -0,0 +1,45 @@
+--- gen-go/agent/metrics/v1/BUILD.bazel.orig	2019-04-30 11:03:46.000000000 -0700
```

Comment:
Could you clarify why this patch is needed? Is this something that should also be upstreamed?